components/AddToCalendar/src/App.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div id="app">
+    <AddToCalendar :event="calEvent" style="display:inline-block;" v-on:click.stop=""/>
+  </div>
+</template>
+
+<script>
+  import AddToCalendar      from './components/'
+  import calEvent           from './mock-event.json'
+
+  export default {
+    name: 'app',
+    data:function(){
+      return{
+        calEvent:calEvent
+    }},
+    components: {
+      AddToCalendar
+    }
+  }
+
+</script>
```

Comment:
can we follow consistent naming convention? folder name, file name should at least use camel case, I like - seperation with all lowercase

Reply:
Yes, I was not sure which to use when I created that.  Components usually are like class names.

Reply:
Kebab case
http://wiki.c2.com/?KebabCase

Reply:
@sbilodeau & @blaisefonseca I strongly wish we get consistent with our comments and suggestions.   I think comments with respect to code style is a waste of our time.  Looking for consistencies in naming and such.  Why consistent with names but not indenting or semi-colons for example?  Consistency with each serves the same purpose.  Why are consistent names important but indenting?

Consistency has its benefits and I strongly want it.  It's far too much to remember when our own code bases are inconsistent over the years.  Or remembering a syntax styling rule we decided during a discussion.  Too much mental load for me.  I rather use the energy for something else.  That being said can we stop making suggestions on code consistency or implement a team .eslintrc.js in order to save time?

I have indicated before Components have a specific naming, did you guys not remember the discussion?  :-)  I feel I am repeating my self.  All this time could be resolved with a team .eslintrc.js.  

https://vuejs.org/v2/style-guide/#Base-component-names-strongly-recommended
https://angular.io/guide/styleguide
https://hackernoon.com/structuring-projects-and-naming-components-in-react-1261b6e18d76

I would also request that we stop using directives as components.  Angularjs has components. https://www.linkedin.com/pulse/angularjs-1x-directives-vs-components-sachila-ranawaka scan to para: "No more Dom Manipulation"   - will continue this in slack.

Reply:
we used camelCase and kebab-case since we started on angular I don't understand what is new here?